### ISSUE TITLE
openshift_hostname is removed 

### ISSUE THREAD

**Issue by cmoulliard** [snowdrop/openshift-infra#118]
When we execute this command `ansible-playbook -i inventory/cloud_host openshift-ansible/playbooks/prerequisites.yml` using openshift-ansible 3.11 release then we get this error

```
TASK [Run variable sanity checks] ***************************************************************************************************************************************************************************************************************************************
fatal: [192.168.99.50]: FAILED! => {"msg": "last_checked_host: 192.168.99.50, last_checked_var: openshift_master_identity_providers;Found removed variables: openshift_hostname is replaced by Removed: See documentation; "}

```

**Comment by cmoulliard** [comment]
I removed the `openshift_hostname` var from the cloud inventory and the error is nor longer there 

```
[OSEv3:children]
masters
nodes
etcd

[OSEv3:vars]
ansible_user=root

public_ip_address=192.168.99.50
host_key_checking=false

openshift_release="3.11.0"

openshift_deployment_type=origin

openshift_additional_repos=[{'id': 'centos-okd-ci', 'name': 'centos-okd-ci', 'baseurl' :'http://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin311/','gpgcheck' :'0','enabled' :'1'}]

openshift_master_cluster_public_hostname=192.168.99.50
openshift_master_default_subdomain=192.168.99.50.nip.io


# To avoid message
# - Available disk space in "/var" (9.5 GB) is below minimum recommended (40.0 GB)
# - Docker storage drivers 'overlay' and 'overlay2' are only supported with 'xfs' as the backing storage, but this host's storage is type 'extfs'
# - Available memory (2.0 GiB) is too far below recommended value (16.0 GiB)
# - Docker version is higher than expected
openshift_disable_check = docker_storage,memory_availability,disk_availability,docker_image_availability,package_version

# Python Interpreter
ansible_python_interpreter=/usr/bin/python

# Enable htpasswd auth
openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider'}]
# htpasswd -nb admin admin
openshift_master_htpasswd_users={'admin': '$apr1$DloeoaY3$nqbN9fQBkyXgbj58buqEM.'}

openshift_node_groups=[{'name': 'node-config-all-in-one', 'labels': ['node-role.kubernetes.io/master=true', 'node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true'], 'edits': [{ 'key': 'kubeletArguments.pods-per-core','value': ['20']}]}]

# Don't install ASB/OAB by default
ansible_service_broker_install=false
ansible_service_broker_image=docker.io/ansibleplaybookbundle/origin-ansible-service-broker:v3.11
ansible_service_broker_registry_whitelist=[".*-apb$"]


# host group for masters
[masters]
192.168.99.50 openshift_public_hostname=192.168.99.50

# host group for etcd, should run on a node that is not schedulable
[etcd]
192.168.99.50 openshift_ip=192.168.99.50

# host group for worker nodes, we list master node here so that
# openshift-sdn gets installed. We mark the master node as not
# schedulable.
[nodes]
192.168.99.50 openshift_node_group_name="node-config-all-in-one" openshift_public_hostname=192.168.99.50
```

**Comment by cmoulliard** [comment]
I think that we fixed this ticket with a recent PR. Correct @geoand ?

**Comment by geoand** [comment]
Let me check

**Comment by geoand** [comment]
Closed via a3ae41d
